openshift_metrics/utils.py

```diff
@@ -290,53 +290,61 @@ def write_metrics_by_namespace(
         metrics["SU_CPU_HOURS"] += su_count_hours
 
         row = [
+            report_month,
             namespace,
             metrics["pi"],
-            report_start_date,
-            report_end_date,
-            str(metrics["_cpu_hours"]),
-            str(metrics["_memory_hours"]),
-            SU_CPU,
+            "", #Invoice Address
+            "", #Institution
+            "", #Institution - Specific Code
             str(metrics["SU_CPU_HOURS"]),
+            SU_CPU,
+            "", #Rate
+            "" #Cost
         ]
         rows.append(row)
 
         if metrics["SU_A100_GPU_HOURS"] != 0:
             row = [
+                report_month,
                 namespace,
                 metrics["pi"],
-                report_start_date,
-                report_end_date,
-                "NA",
-                "NA",
-                SU_A100_GPU,
+                "", #Invoice Address
+                "", #Institution
+                "", #Institution - Specific Code
                 str(metrics["SU_A100_GPU_HOURS"]),
+                SU_A100_GPU,
+                "", #Rate
+                "" #Cost
             ]
             rows.append(row)
 
         if metrics["SU_A10_GPU_HOURS"] != 0:
             row = [
+                report_month,
                 namespace,
                 metrics["pi"],
-                report_start_date,
-                report_end_date,
-                "NA",
-                "NA",
-                SU_A10_GPU,
+                "", #Invoice Address
+                "", #Institution
+                "", #Institution - Specific Code
                 str(metrics["SU_A10_GPU_HOURS"]),
+                SU_A10_GPU,
+                "", #Rate
+                "" #Cost
             ]
             rows.append(row)
 
         if metrics["SU_MOC_GPU_HOURS"] != 0:
             row = [
+                report_month,
                 namespace,
                 metrics["pi"],
-                report_start_date,
-                report_end_date,
-                "NA",
-                "NA",
-                SU_MOC_GPU,
+                "", #Invoice Address
+                "", #Institution
+                "", #Institution - Specific Code
                 str(metrics["SU_MOC_GPU_HOURS"]),
+                SU_MOC_GPU,
```

Comment:
What is MOC GPU? can this be replaced with V100?

Reply:
i added this when nerc didn't have any gpu to test against, so i was testing it against a GPU in MOC cluster.

Reply:
I will take of care of types of GPUs and the associated SUs in a different PR. This PR only deals with headers for now.